src/guifier/fields/baseFields/TextField/TextField.ts

```diff
@@ -53,17 +53,44 @@ export class TextField extends Field {
      * @returns {HTMLElement} html element object
      */
     public draw (): HTMLElement {
-        // drawing the input
-        const inputElement = document.createElement('input')
-        inputElement.type = 'text'
-        inputElement.classList.add('guifierTextField')
-        if (this.property._value !== '') {
-            inputElement.value = this.property._value
+        let inputElement: HTMLInputElement | HTMLTextAreaElement
+
+        // Auto-detect multiline if the value contains line breaks
+        const hasLineBreaks = typeof this.property._value === 'string' && this.property._value.includes('\n')
+
+        if (hasLineBreaks) {
+            // drawing the textarea for multiline text
+            const textareaElement = document.createElement('textarea')
+            textareaElement.classList.add('guifierTextField', 'guifierTextFieldMultiline')
+            if (this.property._value !== '') {
+                textareaElement.value = this.property._value
+            }
+            textareaElement.placeholder = 'Multiline Text Field'
+            
+            // auto-resize functionality for textarea
+            const autoResize = (): void => {
+                textareaElement.style.height = 'auto'
+                textareaElement.style.height = `${Math.max(textareaElement.scrollHeight, 45)}px`
+            }
```

Comment:
the autoResize functionality creates an overflow
<img width="234" height="114" alt="image" src="https://github.com/user-attachments/assets/a4566961-516d-4076-9844-632da2016f15" />
apparently this part `${Math.max(textareaElement.scrollHeight, 45)}px` generates a smaller height the actual text height that exists in the UI
but we can just give it a reasonable min-height and that will be enough